austation/code/modules/mob/living/blood.dm

```diff
@@ -0,0 +1,304 @@
+/****************************************************
+				BLOOD SYSTEM
+****************************************************/
+
+/mob/living/carbon/human/proc/suppress_bloodloss(amount)
+	if(bleedsuppress)
+		return
+	else
+		bleedsuppress = TRUE
+		addtimer(CALLBACK(src, .proc/resume_bleeding), amount)
+
+/mob/living/carbon/human/proc/resume_bleeding()
+	bleedsuppress = 0
+	if(stat != DEAD && bleed_rate)
+		to_chat(src, "<span class='warning'>The blood soaks through your bandage.</span>")
+
+
+/mob/living/carbon/monkey/handle_blood()
+	if(bodytemperature >= TCRYO && !(HAS_TRAIT(src, TRAIT_HUSK))) //cryosleep or husked people do not pump the blood.
+		//Blood regeneration if there is some space
+		if(blood_volume < BLOOD_VOLUME_NORMAL)
+			blood_volume += 0.1 // regenerate blood VERY slowly
+			if(blood_volume < BLOOD_VOLUME_OKAY)
+				adjustOxyLoss(round((BLOOD_VOLUME_NORMAL - blood_volume) * 0.02, 1))
+
+// Takes care blood loss and regeneration
+/mob/living/carbon/human/handle_blood()
+
+	if(NOBLOOD in dna.species.species_traits)
+		bleed_rate = 0
+		return
+
+	if(bodytemperature >= TCRYO && !(HAS_TRAIT(src, TRAIT_HUSK))) //cryosleep or husked people do not pump the blood.
+
+		//Blood regeneration if there is some space
+		if(blood_volume < BLOOD_VOLUME_NORMAL && !HAS_TRAIT(src, TRAIT_NOHUNGER))
+			var/nutrition_ratio = 0
+			switch(nutrition)
+				if(0 to NUTRITION_LEVEL_STARVING)
+					nutrition_ratio = 0.2
+				if(NUTRITION_LEVEL_STARVING to NUTRITION_LEVEL_HUNGRY)
+					nutrition_ratio = 0.4
+				if(NUTRITION_LEVEL_HUNGRY to NUTRITION_LEVEL_FED)
+					nutrition_ratio = 0.6
+				if(NUTRITION_LEVEL_FED to NUTRITION_LEVEL_WELL_FED)
+					nutrition_ratio = 0.8
+				else
+					nutrition_ratio = 1
+			if(satiety > 80)
+				nutrition_ratio *= 1.25
+			adjust_nutrition(-nutrition_ratio * HUNGER_FACTOR)
+			blood_volume = min(BLOOD_VOLUME_NORMAL, blood_volume + 0.5 * nutrition_ratio)
+
+		// austation begin -- Adding Bloodsuckers by Coolsurf6
+		if (HAS_TRAIT(src, TRAIT_NOPULSE)) // Bloodsuckers don't need to be here.
+			return
+		// austation end
```

Comment:
I am looking at this from time to time and please remove austation/code/modules/mob/living/blood.dm, you already made appropriate edits to code/modules/mob/living/blood.dm and austation/code/modules/mob/living/blood.dm isn't even on defines

Reply:
Ahhh, yeah that was when I was trying to do modularizing. I thought you had to jsut take a copy of the file, add the change and define it. That way it would use that one isntead of the other.

Definitely wouldn't work so I removed it from the defines. How the fuck did I not delete it already?